core/src/main/java/io/servicecomb/core/transport/TransportManager.java

```diff
@@ -36,12 +39,18 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(TransportManager.class);
 
     @Inject
-    private List<Transport> transportList;
+    private List<Transport> transports;
 
     private Map<String, Transport> transportMap = new HashMap<>();
 
+    public void setTransports(List<Transport> transports) {
+        this.transports = transports;
+    }
+
     public void init() throws Exception {
-        for (Transport transport : transportList) {
+        buildTransportMap();
+
+        for (Transport transport : transportMap.values()) {
             transportMap.put(transport.getName(), transport);
```

Comment:
yes
unused code, and will not cause problem
when i fetch and rebase, delete this line, then i can not push, git said that: rejected ..... (stale info)

i will delete this line in next PR.